spec/haskell/src/SEL4/Object/Structures.lhs

```diff
@@ -501,6 +501,9 @@ Various operations on the free index of an Untyped cap.
 
 > data TcbFlag = FpuDisabled
 
+> numFlags :: Int
+> numFlags = 1
```

Comment:
this is a confusing name; given that it's called TcbFlag above it should be `numTcbFlags`

Reply:
This ended up not being needed and has been removed.